module/VuFind/src/VuFind/Connection/ExternalVuFind.php

```diff
@@ -43,9 +43,11 @@
  */
 class ExternalVuFind implements
     \VuFindHttp\HttpServiceAwareInterface,
+    \VuFind\Http\CachingDownloaderAwareInterface,
     LoggerAwareInterface
 {
     use \VuFindHttp\HttpServiceAwareTrait;
+    use \VuFind\Http\CachingDownloaderAwareTrait;
```

Comment:
If you're going to rely exclusively on the CachingDownloader for retrieval, I think you can remove the dependency on HttpServiceAwareInterface/Trait.

Reply:
Thanks, fixed.